README.md

```diff
@@ -31,6 +31,9 @@ Currently supports (most) service brokers for the following:
 
 ## Usage
 
+> **Note**
+> If you are using this tool to connect to a service on cloud.gov, your space must be configured with `restricted-egress`. Do this by running `cf bind-security-group trusted_local_networks_egress ORG --space SPACE` with your organization and space. Skipping this step will result in a `connection refused` error. For more, see [cloud.gov: Controlling egress traffic](https://cloud.gov/docs/management/space-egress/).
```

Comment:
```suggestion
> If you are using this tool to connect to a service on cloud.gov, your space must be configured with the `trusted_local_networks_egress` security group. Do this by running `cf bind-security-group trusted_local_networks_egress ORG --space SPACE` with your organization and space. Skipping this step will result in a `connection refused` error. For more, see [cloud.gov: Controlling egress traffic](https://cloud.gov/docs/management/space-egress/).
```

I find the `restricted-egress` name confusing because it's not a real security group, it's just an alias